tools/mod/go.mod

```diff
@@ -1,6 +1,6 @@
 module go.etcd.io/etcd/tools/v3
 
-go 1.22.4
```

Comment:
I found that the dependency that is generating this is: `github.com/alexfalkowski/gocovmerge` (https://github.com/alexfalkowski/gocovmerge/blob/e4976baff208864a31dd6ef35ea755b0cef63030/go.mod#L3). I'll open an issue/pull request in their repository, so we don't have this issue anymore.

Reply:
Opened alexfalkowski/gocovmerge#87.

Reply:
Merged in lightning speed :D! Thanks @ivanvc

Reply:
Release done by gocovmerge https://github.com/alexfalkowski/gocovmerge/releases/tag/v1.3.12
Will add a new commit to address this

Reply:
@ivanvc https://github.com/etcd-io/etcd/pull/18418/commits/39d3e25f73e4d4456c32074c66a7b4243437b2f3